runtime/compiler/trj9/ilgen/J9IlGeneratorMethodDetails.cpp

```diff
@@ -35,12 +35,14 @@
 #include "ilgen/J9ByteCodeIlGenerator.hpp"
 #include "trj9/env/VMJ9.h"
 
+
 namespace TR
 {
 
-IlGeneratorMethodDetails & IlGeneratorMethodDetails::operator=(const TR::IlGeneratorMethodDetails & other)
+IlGeneratorMethodDetails *
+IlGeneratorMethodDetails::clone(TR::IlGeneratorMethodDetails &storage, const TR::IlGeneratorMethodDetails & other)
    {
-   return self()->J9::IlGeneratorMethodDetailsConnector::operator=(other);
```

Comment:
Just curious why you put this particular forwarding function into the cpp file, which means little chance it will ever be inlined...?